README.rst

```diff
@@ -384,6 +384,10 @@ TMT_SHOW_TRACEBACK
     traceback is not printed out. When ``TMT_SHOW_TRACEBACK`` is
     set to any string except ``0``, traceback would be printed out.
 
+TMT_TERM_WIDTH
```

Comment:
I'm not really sure about the abbreviations. Would you be ok with `TMT_TERMINAL_WIDTH` or `TMT_OUTPUT_WIDTH` as well?

Reply:
Fixed in c6285af6. Although, `TERM` is quite easy to notice pretty much everywhere in unix lands :)

Reply:
Thanks.